index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons"
+  rel="stylesheet">
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200" />
+  <link href="https://fonts.googleapis.com/icon?family=Material+Icons"
+  rel="stylesheet">
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/devicons/devicon@v2.15.1/devicon.min.css">
+ 
+  <link rel="stylesheet" href="path/to/font-awesome/css/font-awesome.min.css">
+    <title>mobile skeleton</title>
+</head>
+<body>
+
+    <header>
+        <nav>
+
+            <a href="#"><h2 id="welcome">Welcome</h2></a>
+            <div class="nav-links">
+            <i id="menu" class="material-symbols-outlined">menu</i>
+        </div>
+        </nav>
+        <div class="headline">
+            <h1 class="black">Hey there. I’m Jemutai.<br><span id="green">I’m a software developer</span>
+                </h1>
+                <p class="text"> 
+                    I can help you build a product , feature or website Look through some of my work and<br> experience! If you like what you see and have a project you need coded,<br>
+                   don’t hestiate to contact me.
+                </p>
+                <ul class="social-icons">
+                    <li>
+                        <a href="#"><img src="images/Vector.png" alt="social media icons"></a>
+                    </li>
+                    <li>
+                        <a href="#"><img src="images/Linkedinicon.png" alt="social media icons"></a>
+                    </li>
+                    <li>
+                        <a href="#"><img src="images/Vector1.png" alt="social media icons"></a>
+                    </li>
+                    <li>
+                        <a href="#"><img src="images/Vector2.png" alt="social media icons"></a>
+                    </li>
+                    <li>
+                        <a href="#"><img src="images/Vector3.png" alt="social media icons"></a>
+                    </li>
+                </ul>
+
+        </div>
```

Comment:
- [ ] As this is the beginning of a new block or section of the project, it would be better to make your markup semantically correct by using a ```<section>``` instead of a ```<div>```.Have a look at [this awesome resource](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) to learn when to use or not to use the <section>.
- [ ] Also, make this section not to be inside the ```<header>``` tag. Make the <header> only have the logo and the hamburger meNu. The rest let them be in the <section> tag